examples/headsail-bsp/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 
 pub mod sprintln;
 pub mod uart;
+pub mod timer_unit;
```

Comment:
Since there may be several timers on the platform, an argument could be made that being very specific with the name might be a good idea, e.g. `pulp_timer_unit`. But after reconsidering, I do like `timer_unit` here more. I think its the perfect compromise of specificity and briefness.